domains/test/service/src/keyring.rs

```diff
@@ -0,0 +1,45 @@
+//! Set of test accounts.
+use fp_account::AccountId20;
+use sp_core::ecdsa::{Pair, Public, Signature};
+use sp_core::{ecdsa, keccak_256, Pair as PairT};
+
+#[allow(missing_docs)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum Keyring {
+    Alice,
+    Bob,
+    Charlie,
+    Dave,
+    Eve,
+    Ferdie,
+    One,
+    Two,
+}
+
+impl Keyring {
+    /// Sign `msg`.
+    pub fn sign(self, msg: &[u8]) -> Signature {
+        let msg = keccak_256(msg);
+        self.pair().sign_prehashed(&msg)
+    }
+
+    /// Return key pair.
+    pub fn pair(self) -> Pair {
+        ecdsa::Pair::from_string(self.to_seed().as_str(), None).unwrap()
```

Comment:
Not a blocker but this will not work all kinds of domains. So we would need to re think this at a later time

Reply:
Right, the domain test service is not generic over the signing scheme, will leave a TODO for now.